docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,340 @@
+"""
+
+.. _lightly-tutorial-active-learning-detectron2:
+
+Tutorial 4: Active Learning using Detectron2 on Comma10k
+=========================================================
+
+Active learning is a process of using model predictions to find a new set of
+images to annotate. The images are chosen to have maximal impact on the model
+performance. In this tutorial, we will use a pre-trained object detection model
+to do active learning on a completely unlabeled set of images.
+
+
+In machine learning, we often don't train a model from scratch.
+Instead, we start with an already pre-trained model. For object detection tasks,
+a common pre-training dataset is MS COCO consisting of over 100'000 images
+containing 80 different classes. Our goal is to take an MS COCO pre-trained
+model and optimize it for an autonomous driving task where we only care about
+pedestrians and cars (two classes present within MS COCO). We will proceed as
+follows: First, we will use the pre-trained model to make predictions on our
+task dataset (Comma10k) which has been collected for auonomous driving. Then,
+we use the predictions, self-supervised learning, and active learning with the
+lightly framework to find the 100 most informative images on which we can
+finetune our model.
+
+
+
+
+This tutorial is available as a
+`Google Colab Notebook <https://colab.research.google.com/drive/1r0KDqIwr6PV3hFhREKgSjRaEbQa5N_5I?usp=sharing>`_
+
+In this tutorial you will learn:
+
+*   how to use Lightly Active Learning together with the
+    `detectron2 <https://github.com/facebookresearch/detectron2>`_ framework
+    for object detection
+*   how to use the Lightly Platform to inspect the selected samples
+*   how to download the selected samples for labeling
+
+The tutorial will be divided into
+the following steps. 
+
+#. Installation of detectron2 and lightly
+#. Run predictions using a pretrained model
+#. Use lightly to compute active learning scores for the predictions
+#. Use the Lightly Platform to understand where our model struggles
+#. Select the most valuable 100 images for annotation
+
+Requirements
+------------
+- Make sure you have OpenCV installed to read and the preprocess the images.
+  You can install the framework using the following command:
+
+.. code::
+  
+   pip install opencv-python
+
+
+- Make sure you have the detectron2 framework installed on your machine. Check out
+  the `detectron2 installation documentation <https://detectron2.readthedocs.io/en/latest/tutorials/install.html>`_
+
+- In this tutorial we work with the comma10k dataset. The dataset consists of
+  10'000 images for autonomous driving and is available
+  `here on GitHub <https://github.com/commaai/comma10k>`_
+  We can download the dataset using `git clone`. We save the dataset locally
+  to `/datasets/`
+
+.. code::
+
+    git clone https://github.com/commaai/comma10k
```

Comment:
If we save the dataset locally to `/datasets/`, you could add the save location directly to the git clone, e.g.
```bash
git clone https://github.com/commaai/comma10k /datasets/
```

Reply:
I would leave it up for the user. Not sure everyone likes to store things into `/datasets`. I can add a line to clarify, that this is just the setup we have.